app/assets/javascripts/uploadcare/settings.coffee

```diff
@@ -201,7 +203,18 @@ uploadcare.namespace 'settings', (ns) ->
 
     if settings.validators
       settings.validators = settings.validators.slice()
-
+    
+    if settings.previewProxy and not settings.previewUrlCallback
```

Comment:
There is an issue: If you set `previewProxy` on some level (UPLOADCARE_PREVIEW_PROXY, for example), it will define `settings.previewUrlCallback`. So, on next level (for example, `data-preview-proxy` attribute) this setting will be just skipped because `settings.previewUrlCallback` is already here.

Reply:
@homm it is not problem that `settings.previewUrlCallback` is not overridden. But there was a problem that `previewUrlCallback` accessing `previewProxy` from settings object from local closure instead of call context. I fixed it.